packages/genesys-spark-components/src/components/beta/gux-time/tests/__snapshots__/gux-time.spec.ts.snap

```diff
@@ -2467,7 +2467,7 @@ exports[`gux-time #render using non regional dates #render different time zones
 exports[`gux-time #render using non regional dates #render different time zones should work as expected for "Antarctica/Vostok" 1`] = `
 <gux-time-beta datetime="2022-07-07T12:00:00.000Z" format="full" time-zone="Antarctica/Vostok">
   <mock:shadow-root>
-    6:00:00 PM Vostok Time
+    7:00:00 PM Vostok Time
```

Comment:
https://timezonedb.com/time-zones/Antarctica/Vostok
After Monday, 18 December, 2023 01:59:59 AM: Clocks were moved backward to become Monday, 18 December, 2023 12:00:00 AM